api/src/main/kotlin/com/gmtkgamejam/bot/DiscordBot.kt

```diff
@@ -4,26 +4,43 @@ import com.gmtkgamejam.Config
 import kotlinx.coroutines.future.await
 import org.javacord.api.DiscordApi
 import org.javacord.api.DiscordApiBuilder
+import org.javacord.api.entity.channel.ServerTextChannel
+import org.javacord.api.entity.message.Message
 import org.javacord.api.entity.message.MessageBuilder
 import org.javacord.api.entity.user.User
 import org.javacord.api.exception.DiscordException
 import org.javacord.api.exception.MissingPermissionsException
-import java.util.concurrent.CompletableFuture
 
-class ContactPermissionsCheckerBot {
+class DiscordBot {
 
     private lateinit var api: DiscordApi
 
+    private lateinit var channel: ServerTextChannel
+
     init {
         val token = Config.getString("bot.token")
         val builder = DiscordApiBuilder().setToken(token)
         try {
             api = builder.login().join()
+
+            // This is horrific, but it works for now!
+            channel = api.channels.filter { it.asServerChannel().get().name == "bot-spam" }[0].asServerTextChannel().get()
```

Comment:
Context for review: #bot-spam is the testing channel in the Team Finder Sandbox server - when we want this live and working, we'll create a new channel in the proper GMTK server